sdk/lib/cmlib/hivewrt.c

```diff
@@ -135,7 +135,8 @@ HvpWriteLog(
      * Now calculate the bitmap and buffer sizes to hold up our
      * contents in a buffer.
      */
-    BitmapSize = ROUND_UP(sizeof(ULONG) + RegistryHive->DirtyVector.SizeOfBitMap / 8, HSECTOR_SIZE);
+    BitmapSize = ROUND_UP(sizeof(ULONG) + RegistryHive->DirtyVector.SizeOfBitMap, HSECTOR_SIZE);
```

Comment:
Could you clarify this change? `SizeOfBitMap` is the number of _**bits**_  in the bitmap, while this `BitmapSize` variable that is being calculated is _**bytes**_.
Other usages of such `SizeOfBitMap` usually divide by 8 whenever a corresponding size in bytes is needed, see for example:
```
sdk/lib/cmlib/hivebin.c	
  79         if (BitmapSize > RegistryHive->DirtyVector.SizeOfBitMap / 8)
  85             if (RegistryHive->DirtyVector.SizeOfBitMap > 0)
  90                               RegistryHive->DirtyVector.SizeOfBitMap / 8);
```

Reply:
I added a comment that explains this in L171 below -- you're right, it might be helpful up here, too.
Our in-memory dirty vector tracks HBLOCK_SIZE (4096 bytes) per bit. The on-disk vector tracks 512 bytes per bit. So for each dirty page we write 8 bits of 1s into the log file's dirty vector. So I suppose this is a `* 8 / 8` in reality. I'll add a comment to clarify.